.github/workflows/stale.yml.jinja

```diff
@@ -0,0 +1,48 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+    - cron: "0 12 * * 0"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+
+      # 120+30 day stale policy for PRs
+      # * Except PRs marked as "don't stale", "help wanted" or "needs review"
+      - name: Stale PRs policy
+        uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          exempt-pr-labels: "don't stale,help wanted,needs review"
```

Comment:
I'd just use `no-stale` (I hope I'm wrong, but that `'` may be problematic somewhere).

IMHO if "help wanted" but nobody wants to help, it should stale. The same if it "needs review" but nobody reviews. If we have some PRs that need review for v7 and nobody wants them anymore, they should fade away.

Reply:
I've just changed it to `no stale` :) Spaces are ok as far as I can see, by checking other project's implementation of this action.

As for `help wanted` and `needs review`, yes indeed :) I thought maybe we should start with a less intrusive approach, but in fact we already have `no stale` for this :+1:

Reply:
Just a question, would it have sense to add the Ready to Merge or the Approved on the exempt labels?

I mean, sometimes, an approved PR is kept there for months until some PSC runs the bot, it is not usual on most of repositories, but it might happen on unusual repos.

Reply:
Hmmm I think I wouldn't add an exception for those.
Some are labeled as "Approved" or "Ready to merge" while in fact there are unattendeed blocking suggestions. I think we should treat them like any other.

Also having this post the stale message is a way to move things forward if they were forgotten.. that is, if the public interest in the PR remains.

Reply:
Yes, I agree with @ivantodorovich.

Besides, our git bot will send some reminders when a PR is in such state, so that use case is already covered. If nobody attends that still, then probably a human should ping another human, or the PR should stale/die just like any other.